src/__tests__/linter.test.ts

```diff
@@ -144,11 +144,11 @@ describe('linter', () => {
   });
 
   test('should not report anything for disabled rules', async () => {
-    await spectral.loadRuleset('spectral:oas3');
-    const { rules: oas3Rules } = await readRuleset('spectral:oas3');
+    await spectral.loadRuleset('spectral:oas');
+    const { rules: oasRules } = await readRuleset('spectral:oas');
     spectral.setRules(
-      mergeRules(oas3Rules, {
-        'valid-example-in-schemas': 'off',
+      mergeRules(oasRules, {
+        'oas3-valid-schema-example': 'off',
         'components-schema-description': -1,
```

Comment:
Let's change this rule to something else since it's removed.

Reply:
@P0lip Replaced with `operation-2xx-response`